### DISCUSSION TITLE
How to persist generated mockdata?

### DISCUSSION THREAD

**Discussion by tobiasqueck** [SAP/open-ux-odata#194] [Q&A] [answered]
I am heavily using the `generateMockData: true` feature with new projects since I don't need more specific mockdata for many of my tests. However, sometimes, I would like to slightly modify it for my needs. Is there a way to persist the generated mockdata and then edit it?

**Comment by carwelsh** [comment, marked as answer]
SAP Fiori Tools is an extension pack that simplifies the development of SAP Fiori element applications. It contains a tool called the Data Editor which will enable you not only to generate and store mock data for your back end service, it will also allow you to edit this data so it fits your needs. This will enable you to create realistic mock data.
